test/test-trace-web-frameworks.ts

```diff
@@ -0,0 +1,323 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Nit. Should be Google "LLC" now. I think only newly added files can be updated.

Reply:
Done